.vscodeignore

```diff
@@ -8,3 +8,4 @@ test/
 **/*.map
 **/tslint.json
 build/**/*
+Documentation/
```

Comment:
Can we use either `documentation` or `docs` as the folder name instead? JS projects usually use camel casing in names and short names in general

Reply:
Moved to docs. Sorry for the forced push (I messed up)